masterlist.yaml

```diff
@@ -4056,6 +4056,15 @@ plugins:
       - crc: 0x8D567201
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'The Temple of Medusa.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/37505/' ]
+    tag: [ EnchantmentStats ]
```

Comment:
That edit looks like a potential wild edit. At the very least, it probably doesn't warrant a tag.

```suggestion
```